### PR TITLE
First wave of changes

### PR DESCRIPTION
**Pull request by cjvanlissa**

### CODE REVIEW

File: manuscript.Rmd [hunk 1]
```diff
@@ -42,19 +42,19 @@ simplesummary: |
   Reproducibility has long been considered integral to the scientific method.
   An analysis is considered reproducible if an independent person can obtain the same results from the same data.
   Until recently, detailed descriptions of methods and analyses were the primary instrument for ensuring scientific reproducibility.
-  Technological advancements now enable scientists to achieve a more comprehensive standard;
-  a standard where that allows anyone to access a digital research repository and reproduce all computational steps from raw data to final report including all relevant statistical analyses, with a single command.
+  Technological advancements now enable scientists to achieve a more comprehensive standard
+  that allows anyone to access a digital research repository and reproduce all computational steps from raw data to final report, including all relevant statistical analyses, with a single command.
```
Comment (brandmaier) [inline manuscript.Rmd:46]:
```suggestion
  that allows anyone to access a digital research repository and reproduce all computational steps from raw data to final report, including all statistical analyses, with a single command.
```

**Review by brandmaier — CHANGES_REQUESTED** [review]

**Comment by aaronpeikert** [comment]
Things I thought about while incorporating changes:

> CJ: Aaron: I don't think the key point is that users preregister the presentation of their results (tables, figures), but rather that they preregister the analysis!!!

Of course, the analysis is the main thing, but there are RDF in the presentation of results. E.g. some researchers might decide in light of small effect size to not report the effect size and concentrate on the statistical significance.
I found it interesting to learn that the FDA expects shells for tables and graphics for clinical trials for those reasons.

> Collaborating via `Git` requires the repository to be uploaded somewhere.

> CJ This is not true

How is this not true? Sure a git repo can be placed anywhere but you require shared access to that place.

**Comment by cjvanlissa** [comment]
> Of course, the analysis is the main thing, but there are RDF in the presentation of results. E.g. some researchers might decide in light of small effect size to not report the effect size and concentrate on the statistical significance.
I found it interesting to learn that the FDA expects shells for tables and graphics for clinical trials for those reasons.

OK, but I think that should be explicitly clarified.

> Collaborating via Git requires the repository to be uploaded somewhere.

People could work on a shared drive, or email each other the repository.. it would be more accurate to say that collaboration is "facilitated by" using a remote git host

**Comment by aaronpeikert** [comment]
@cjvanlissa Hey Casper, thanks for the input, but did you realize that the paper is out?

**Comment by cjvanlissa** [comment]
Of course, but I hope that is not the end of the conversation! ;)

**Pull request merged**